src/util/request.js

```diff
@@ -5,11 +5,15 @@ import testMakerResponse from './testMakerResponse';
 import testSearchResponse from './testSearchResponse';
 
 // TODO make this a config file
-const testQuery = '/test/motif';
-const searchQuery = '/align/activesite';
-
-const useForm = (callback) => {
-  const [values, setValues] = useState({});
+const testQueryURL = 'http://localhost:8080/test/motif';
+const searchQueryURL = 'http://localhost:8080/align/activesite';
+const dockRequestURL = 'http://localhost:8080/dock/dockligand';
+const dockingMoleculeFileRetrievalURL = 'http://localhost:8080/dock/retrievefile';
+const exportDockingInfoURL = 'http://localhost:8080/dock/exportLigands';
+const ligandLibraryURL = 'http://localhost:8080/ligands';
```

Comment:
These don't need the domain or port. You can set them back to just `/test/motif`, `/align/activesite`, and everything else.